ansys/mapdl/core/post.py

```diff
@@ -408,6 +937,17 @@ def _all_nnum(self):
         self._mapdl.cmsel("S", "__TMP_NODE__", "NODE")
         return nnum
 
+    @property
+    @supress_logging
+    def _all_enum(self):
+        self._mapdl.cm("__TMP_ELEM__", "ELEM")
+        self._mapdl.allsel()
+        nnum = self._mapdl.get_array("ELEM", item1="NLIST").astype(np.int32)
+        if nnum[0] == -1:
+            nnum = self._mapdl.get_array("ELEM", item1="NLIST").astype(np.int32)
```

Comment:
>         if nnum[0] == -1:
>             nnum = self._mapdl.get_array("ELEM", item1="NLIST").astype(np.int32)

What is the purpose of this??

Reply:
Weird edge case where sometimes we get back a negative number at the start of the element list.

I've added a note as a comment.